frontend/app/models/encryptable-transfer.js

```diff
@@ -0,0 +1,56 @@
+import Encryptable from "./encryptable";
+import { attr } from "@ember-data/model";
+import { inject as service } from "@ember/service";
+
+export default class EncryptableTransfer extends Encryptable {
+  @attr file;
+  @service router;
+  @service fetchService;
+
+  async save() {
+    if (this.isDeleted) {
+      return super.save();
+    }
+    const targetUrl = `/api/encryptables_transfer`;
+    const receiverId = await this.receiver.get("id");
+    let opts;
+
+    if (this.description === undefined && this.file === undefined) {
```

Comment:
extract to getter with meaningful naming of this condition

Reply:
Check, if this is a good check, if not replace it and extract check to a getter